Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM talis/ubuntu:1404-latest
```

Comment:
14.04 is unsupported now - would 18.04 not be better at this point?

Reply:
I do prefer ubuntu over alpine for this development containers though. :-)

Reply:
I want to keep the dev environment the same as what we've deployed for echo since im debugging an issue on there.